content/zh-cn/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -129,13 +125,14 @@ causing failed requests when the average load is acceptably low.
 ## 概念    {#concepts}
 
 APF 特性包含几个不同的功能。
-传入的请求通过 _FlowSchema_ 按照其属性分类，并分配优先级。
+传入的请求通过 **FlowSchema** 按照其属性分类，并分配优先级。
 每个优先级维护自定义的并发限制，加强了隔离度，这样不同优先级的请求，就不会相互饿死。
-在同一个优先级内，公平排队算法可以防止来自不同 _flow_ 的请求相互饿死。
+在同一个优先级内，公平排队算法可以防止来自不同 **流（Flow）** 的请求相互饿死。
```

Comment:
```suggestion
在同一个优先级内，公平排队算法可以防止来自不同**流（Flow）**的请求相互饿死。
```

Reply:
I'm gonna delete these spaces and see how the output is rendered.